.evergreen/install-dependencies.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+install_extension ()
+{
+   rm -f ${PHP_PATH}/lib/php.ini
+
+   if [ "x${DRIVER_BRANCH}" != "x" ] || [ "x${DRIVER_REPO}" != "x" ]; then
+      CLONE_REPO=${DRIVER_REPO:-https://github.com/mongodb/mongo-php-driver}
+      CHECKOUT_BRANCH=${DRIVER_BRANCH:-master}
+
+      echo "Compiling driver branch ${CHECKOUT_BRANCH} from repository ${CLONE_REPO}"
+
+      mkdir -p /tmp/compile
+      git clone ${CLONE_REPO} /tmp/compile/mongo-php-driver
+      cd /tmp/compile/mongo-php-driver
+
+      git checkout ${CHECKOUT_BRANCH}
+      git submodule update --init
+      phpize
+      ./configure --enable-mongodb-developer-flags
+      make all -j20 > /dev/null
+      make install
+   elif [ "x${DRIVER_VERSION}" != "x" ]; then
+      echo "Installing driver version ${DRIVER_VERSION} from PECL"
+      pecl install -f mongodb-${DRIVER_VERSION}
+   else
+      echo "Installing latest driver version from PECL"
+      pecl install -f mongodb
+   fi
+
+   sudo cp ${PROJECT_DIRECTORY}/.evergreen/config/php.ini ${PHP_PATH}/lib/php.ini
+}
+
+DIR=$(dirname $0)
+# Functions to fetch MongoDB binaries
+. $DIR/download-mongodb.sh
+OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+
+get_distro
+
+# See .evergreen/download-mongodb.sh for most possible values
+case "$DISTRO" in
+   cygwin*)
+      echo "Install Windows dependencies"
+      ;;
+
+   darwin*)
+      echo "Install macOS dependencies"
+      ;;
+
+   linux-rhel*)
+      echo "Install RHEL dependencies"
+      ;;
```

Comment:
Is this only empty because you're ignoring RHEL in the test matrix, or are no additional dependencies needed? I assume this might have been copied from PHPC?

Reply:
This is the default file as copied over from the tools repository. We can remove this if we don't want dead code from the basic file here.

Reply:
No objections.